src/libsync/propagatorjobs.cpp

```diff
@@ -13,6 +13,7 @@
  * for more details.
  */
 
+#include "configfile.h"
```

Comment:
Please don't add a dependency between the config file and the sync engine!
Add the option in the `SyncOption`  struct